src/providers/reader.spec.ts

```diff
@@ -153,7 +154,7 @@ describe('Providers → Reader', () => {
 
 				const entry = tests.getFileEntry(false /** dot */);
 
-				const expected: string = path.join(process.cwd(), 'fixtures/file.txt');
```

Comment:
Are results always converted to forward slashes? This makes matching slashes paths that are returned hard since Node will normalize slashes in Windows as backslashes.